demo/demo_quantized_simulation_letters_debug.py

```diff
@@ -0,0 +1,369 @@
+import taichi as ti
```

Comment:
Let's remove this file from this commit :-)

Reply:
guess I would move it to tests temporarily